docs/index.asciidoc

```diff
@@ -5,6 +5,8 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
 include::overview.asciidoc[]
 
+include::compatibility.asciidoc[]
```

Comment:
If you move the compatibility info under `Installation`, then this item can be deleted.
```suggestion
```